src/main/resources/hudson/tasks/test/MetaTabulatedResult/body.jelly

```diff
@@ -24,94 +24,79 @@ THE SOFTWARE.
-          </j:forEach>
-        </tr>
-      </thead>
-      <j:forEach var="f" items="${it.failedTests}" varStatus="i">
-        <tr>
-          <td class="pane no-wrap"><t:failed-test result="${f}" url="${f.getRelativePathFrom(it)}"/></td>
-          <td class="pane no-wrap" style="text-align:right;" data="${f.duration}">
-            ${f.durationString}
-          </td>
-          <td class="pane" style="text-align:right;">
-            <a href="${rootURL}/${f.failedSinceRun.url}">${f.age}</a>
-          </td>
-          <j:forEach var="tablerow" items="${f.testActions}">
-            <st:include it="${tablerow}" page="tablerow.jelly" optional="true"/>
-          </j:forEach>
-        </tr>
-      </j:forEach>
-    </table>
+    <f:section title="${%All Failed Tests}">
+      <t:table items="${it.failedTests}" id="failedtestresult" />
+    </f:section>
   </j:if>
 
   <j:if test="${it.totalCount!=0}">
-    <h2>${%All Tests}</h2>
+    <f:section title="${%All Tests}">
     <table class="jenkins-table sortable" id="testresult">
       <thead>
         <tr>
           <th>${it.childTitle}</th>
-          <th style="width:5em; text-align:right">${%Duration}</th>
-          <th style="width:5em; text-align:right">${%Fail}</th>
-          <th style="width:3em; text-align:right; white-space:nowrap;">(${%diff})</th>
-          <th style="width:5em; text-align:right">${%Skip}</th>
-          <th style="width:3em; text-align:right; white-space:nowrap;">(${%diff})</th>
-          <th style="width:5em; text-align:right">${%Pass}</th>
-          <th style="width:3em; text-align:right; white-space:nowrap;">(${%diff})</th>
-          <th style="width:5em; text-align:right">${%Total}</th>
-          <th style="width:3em; text-align:right; white-space:nowrap;">(${%diff})</th>
+          <th style="width: 8em;">${%Fail}</th>
+          <th style="width: 8em;">${%Skip}</th>
+          <th style="width: 8em;">${%Pass}</th>
+          <th style="width: 8em;">${%Total}</th>
           <j:forEach var="tableheader" items="${it.testActions}">
             <st:include it="${tableheader}" page="${it.childType}tableheader.jelly" optional="true"/>
           </j:forEach>
+          <th style="width: 8em;">${%Duration}</th>
         </tr>
       </thead>
       <tbody>
         <j:set var="prevAll" value="${it.previousResult}" />
         <j:forEach var="p" items="${it.children}">
           <j:set var="prev" value="${prevAll.findCorrespondingResult(p.id)}" />
           <tr>
-            <td class="pane">
+            <td>
               <j:invokeStatic var="encodedSafeName" className="hudson.Util" method="rawEncode">
                 <j:arg value="${p.safeName}" />
               </j:invokeStatic>
-              <a href="${encodedSafeName}/" class="body.jelly"><span style="${prev==null?'font-weight:bold':''}"><st:out value="${p.displayName}" /></span></a>
+              <a href="${encodedSafeName}/" style="font-family: var(--font-family-mono)">
```

Comment:
I'd prefer not using a monospaced font here, because it tends to be wider and then more likely to widen the table so much that it needs horizontal scrolling.